### DISCUSSION TITLE
multiple file api docs possible?

### DISCUSSION THREAD

**Discussion by th3fallen** [PaloAltoNetworks/docusaurus-openapi-docs#526] [Q&A]
Our current api docs are split in to several .yaml files that import each other as needed but when i try to generate with this plugin it fails to resolve some of them in the complete wrong directory.

Is this possible with this plugin or only with https://github.com/rohit-gohri/redocusaurus? because i really love your concept of versioning but if i cant build docs then its not really worth much 😢

**Comment by sserrata** [comment]
Hi @th3fallen, our plugin supports OpenAPI specifications that are split into multiple files ($ref pointers) and it also supports reading in multiple specs from a directory. In fact, we use both in production today. Can you provide more context on the issue you are seeing or a reproduction? Thanks!
